credentials/apps/edx_credentials_extensions/edly_credentials_app/utils.py

```diff
@@ -0,0 +1,81 @@
+import logging
+
+import jwt
+from django.conf import settings
+from django.contrib.sites.shortcuts import get_current_site
+
+from credentials.apps.core.models import SiteConfiguration
+
+LOGGER = logging.getLogger(__name__)
+
+
+def user_has_edx_organization_access(request):
+    """
+    Check if the requested URL site is allowed for the user.
```

Comment:
Update the docstring
```
Check if the user can access the requested URL by verifying its linked edx organization.`

This method checks if the "edx_org_short_name" related to the requested URL's site
matches with the "edx-org" in the "edly_user_info" cookie.
```


Related code in LMS:

```python
def _get_edly_user_info_cookie_string(request):
    """
    Returns JWT encoded cookie string with edly user info.

    Arguments:
        request (HttpRequest): Django request object

    Returns:
        string
    """
    try:
        edly_sub_organization = request.site.edly_sub_org_for_lms
        edly_user_info_cookie_data = {
            'edly-org': edly_sub_organization.edly_organization.slug,
            'edly-sub-org': edly_sub_organization.slug,
            'edx-org': edly_sub_organization.edx_organization.short_name,
            'is_course_creator': auth.user_has_role(
                request.user,
                CourseCreatorRole()
            ) if getattr(request, 'user', None) else False,
        }
        return encode_edly_user_info_cookie(edly_user_info_cookie_data)
    except EdlySubOrganization.DoesNotExist:
        return ''
```

Reply:
Done